### PR TITLE
Feature: Ge tour

### PR DESCRIPTION
**Pull request by Onokaev**
## Overview
Gives a tour of Graph Explorer features for both getting started and advanced experiences. 

### Demo
image
image
A coachmark appears when GE is first loaded. This only appears once

image
Sample teachingbubble

image
After completing the tour, a message like this pops up with a link to the docs

image
If a user opts out of the tour mid-way, this message pops up

image
The Tour button is under the More Actions menu button

image
When the tour is launched from the 'More Actions' button, this dialog pops up.

image
Start a tour by selecting your preferred tour type

image
The tour adds an overlay over the rest of the page to maintain focus on the component being demonstrated

image
A context menu has been added on the response pivots so that you can start the tour from the response components

### Notes
#1140 #1141 
## Testing Instructions
On the more actions button, click Graph Explorer Tour

### CODE REVIEW

**Comment by Onokaev** [comment]
> When the tour dialog pops up, it has the first section selected by default. Let's change this to have no options selected so that a user can choose for themselves which one they'd like.
> 
> We can also have a button that says "start tour" which closes the dialog and starts the tour

Sure. Let me add that

**Comment by Onokaev** [comment]
> image
> 
> This is awesome!
> 
> _question_: How do users know that the items are right-clickable?

I can add a small note on the initial dialog to notify users of this

File: src/app/views/query-response/QueryResponse.tsx [hunk 1]
```diff
@@ -101,16 +104,48 @@ const QueryResponse = (props: IQueryResponseProps) => {
     }
   };
 
+  const selectContextItem = (e: any, item: any, link: any) => {
```
Comment (jobala) [inline src/app/views/query-response/QueryResponse.tsx:107]: Consider using the appropriate type instead of **any**

Comment (Onokaev) [reply src/app/views/query-response/QueryResponse.tsx]: Thank you. 
Fixed!

**Comment by Onokaev** [comment]
Will reopen when feature is planned for

**Pull request closed without merging**